### DISCUSSION TITLE
Welcome to HyperSnips Discussions!

### DISCUSSION THREAD

**Discussion by draivin** [draivin/hsnips#48] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  If you have any doubts about usage, or have snippets to share, please don't be afraid to start a new thread!

**Comment by UtahKemeran** [comment]
Hey, I just wanted to thank you for bringing UltiSnips to VSCode, great work! Unfortunately, it seems that many people don't know your extension yet, I just stumbled across [this](https://github.com/microsoft/vscode/issues/125410) feature request yesterday. Again, thanks a lot for this great extension, I hope it will gain more popularity soon!
